
uint32 drone_id

# velocity, waypoint, take-off, land, emergency land etc.
string msg_type

# either a velocity change or position change
multi_drone_platform/movementFeedback movement
